CSharpLevel_2/Star_2.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace CSharpLevel_2
+{
+    class Star_2 : VisualObject
+    {
+        public Star_2(Point Position, Point Direction, int Size) : base(Position, Direction, new Size(Size, Size))
+        {
+
+        }
+
+        public override void Draw(Graphics g)
+        {
+            Game.__Buffer.Graphics.FillRectangle(Brushes.White, _Position.X, _Position.Y, _Size.Width, _Size.Height);
```

Comment:
```suggestion
            g.FillRectangle(Brushes.White, _Position.X, _Position.Y, _Size.Width, _Size.Height);
```

в данном методе к Вас есть переменная g. Лучше использовать её.